From f05bb8c6578c8dac50bc0a92dd951a6f86690492 Mon Sep 17 00:00:00 2001
From: Enrico Jorns <e.joerns@tu-bs.de>
Date: Mon, 29 Jan 2018 19:27:13 +0100
Subject: [PATCH 3/3] src: caformat.h: do not set selinux flags by default when
 compiled without selinux (#127)

---
 src/caformat.h | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/caformat.h b/src/caformat.h
index 8c6c12b..7abe8c5 100644
--- a/src/caformat.h
+++ b/src/caformat.h
@@ -146,7 +146,9 @@ enum {
                 CA_FORMAT_WITH_SUBVOLUME_RO|
                 CA_FORMAT_WITH_XATTRS|
                 CA_FORMAT_WITH_ACL|
+#if HAVE_SELINUX
                 CA_FORMAT_WITH_SELINUX|
+#endif
                 CA_FORMAT_WITH_FCAPS,
 
         CA_FORMAT_WITH_UNIX = /* Conservative UNIX file properties */
@@ -193,7 +195,9 @@ enum {
                 CA_FORMAT_WITH_SUBVOLUME|
                 CA_FORMAT_WITH_SUBVOLUME_RO|
                 CA_FORMAT_WITH_ACL|
+#if HAVE_SELINUX
                 CA_FORMAT_WITH_SELINUX|
+#endif
                 CA_FORMAT_WITH_FCAPS,
 
         CA_FORMAT_WITH_FUSE = /* All bits that may also be exposed via fuse */
@@ -244,7 +248,9 @@ enum {
                 CA_FORMAT_WITH_SUBVOLUME_RO|
                 CA_FORMAT_WITH_XATTRS|
                 CA_FORMAT_WITH_ACL|
+#if HAVE_SELINUX
                 CA_FORMAT_WITH_SELINUX|
+#endif
                 CA_FORMAT_WITH_FCAPS,
 
         CA_FORMAT_DEFAULT = /* The default set of flags */
-- 
2.18.0

